listling/tests/test_listling.py

```diff
@@ -119,6 +119,9 @@ def test_update_db_version_first(self):
         # Update to version 8
         self.assertEqual([user.id for user in app.lists[1].users()],
                          [user.id for user in reversed(app.users[0:2])])
+        # Update to version 9
+        for l in app.lists.values(): # pylint: disable=no-member
```

Comment:
Ohne den Kommentar frisst es der Linter nicht. Ich checke nicht warum... Behauptet `values()` würde es nicht geben.

Reply:
IIRC this is Pylint bug, it is confused if a class inherits from `Generic`. ;)